cpp/turbodbc/Library/turbodbc/field.h

```diff
@@ -12,7 +12,7 @@ namespace turbodbc {
  * @brief This type represents a single field in a table, i.e., the data associated
  *        with a given row and column
  */
-using field = boost::variant<	long,
+using field = boost::variant<	intptr_t,
```

Comment:
`int64_t` is the correct type since everywhere `SQL_BIGINT` is used for binding to retrieve 64 bit data.